tests/python/direct/test_python_frontend.py

```diff
@@ -633,6 +633,86 @@ def fusion_func(fd: FusionDefinition):
     test_fn(1)
 
 
+def test_cumsum(nvfuser_direct_test):
```

Comment:
Prefer Opinfo testing for new operations.

We have input generators for `reduction`, which can be modified for `scan`.
https://github.com/NVIDIA/Fuser/blob/main/tests/python/opinfo/opinfo_input_generators.py#L1190-L1263

Reply:
Understood. I updated issue #432.